docs/build-your-first-blueprint.md

```diff
@@ -113,7 +113,7 @@ The site should now look like the screenshot below:
 
 ### Resources
 
-The `themeZipFile` defines a [resource](https://wordpress.github.io/wordpress-playground/blueprints-api/resources/)—a reference to an external file required to complete the step. Playground supports different types of resources, including `url`, `wordpress.org/themes`, `wordpress.org/plugins`, `vfs`, or `literal`. The example uses the `wordpress.org/themes` resource, which requires a `slug` identical to the one used in WordPress theme directory:
+The `themeZipFile` defines a [resource](https://wordpress.github.io/wordpress-playground/blueprints-api/resources/) and referrences an external file required to complete the step. Playground supports different types of resources, including `url`, `wordpress.org/themes`, `wordpress.org/plugins`, `vfs`, or `literal`. The example uses the `wordpress.org/themes` resource, which requires a `slug` identical to the one used in WordPress theme directory:
```

Comment:
@bph thank you for the phrasing updates!

Do you mind if I ask where you found the relevant info regarding resource types: 

> Playground supports different types of resources, including `url`, `wordpress.org/themes`, `wordpress.org/plugins`, `vfs`, or `literal`

working on docs...

Reply:
@flexseth https://wordpress.github.io/wordpress-playground/blueprints-api/resources
I wondered if I should make it a bullet list in that paragraph.

Reply:
changed it into bullet list.

Reply:
Take look if that works for you.